### ISSUE TITLE
'System lock' range should be indicated in the status bar

### ISSUE THREAD

**Issue by avvvvve** [musescore/MuseScore#26336]
### Expected behavior
When a system lock is selected, it should say where its start and end measures are, like the following:

`System lock; Start measure: 83; End measure: 97
`

### Current behavior
When a system lock is selected, all the bottom bar says is "System lock".
     <img width="328" alt="Image" src="https://github.com/user-attachments/assets/1cfbb868-34d7-436e-8b0b-7b052f4960aa" />

Other layout elements provide information about where the element is. For example, a range selection looks like this (though the beat value is unnecessary for system locks):
     <img width="454" alt="Image" src="https://github.com/user-attachments/assets/6fad0afd-56c3-4601-930e-b25d44d52d0b" />
